oss-internship-2020/curl/custom_curl/custom_curl.h

```diff
@@ -12,21 +12,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// wrapper for curl library used to implement variadic functions explicitly
+// Wrapper for curl library used to implement variadic methods explicitly
 
 #ifndef CUSTOM_CURL_H
 #define CUSTOM_CURL_H
 
-#include <curl/curlver.h>
-#include <curl/system.h>
 #include <curl/curl.h>
 
-extern "C" CURLcode curl_easy_setopt_ptr(CURL *handle, CURLoption option, void* parameter);
+extern "C" CURLcode curl_easy_setopt_ptr(CURL *handle, CURLoption option, 
```

Comment:
Consistency: Prefer to keep the asterisk `*` on the left side, close to the type. `custom_curl.cc` does this already.

Note that clang-format will count asterisks and then reformat close to the type or to the variable, depending on where the majority was placed before.